sysstat-post-process

```diff
@@ -504,7 +504,52 @@ for my $log_file (sort readdir($dh)) {
             exit;
             }
         }
-    }
+    } elsif ($log_file =~ /^iostat-stdout.txt$/) {
+	my $log_fh = $log_file;
+        open(TMP,"$log_fh") || die "Could not open $log_fh\n";
+        my $separator = $/;
+        $/ = '';
+        my $ymd_timestamp_ms; # Epochtime in milliseconds
+
+        while (my($timestamp, $header, @devicelines) = split('\n', <TMP>)) {
+            foreach my $device_entry (@devicelines) {
+		my %iosample;
+                my $iodev = (split ' ', ${device_entry})[0];
+                (my $unused,
+                $iosample{$iodev}{'reads-sec'},
+                $iosample{$iodev}{'writes-sec'},
+                $iosample{$iodev}{'read-kb-sec'},
+                $iosample{$iodev}{'write-kb-sec'},
+                $iosample{$iodev}{'reads-merged-sec'},
+                $iosample{$iodev}{'writes-merged-sec'},
+                $iosample{$iodev}{'read-pct-merged'},
+                $iosample{$iodev}{'write-pct-merged'},
+                $iosample{$iodev}{'read-wait-avg-ms'},
+                $iosample{$iodev}{'write-wait-avg-ms'},
+                $iosample{$iodev}{'avg-queue-len'},
+                $iosample{$iodev}{'avg-read-req-size'},
+                $iosample{$iodev}{'avg-write-req-size'},
+		my $deprecated,
```

Comment:
What is being deprecated?

Reply:
The value from the 'svctm' field is deprecated and may be removed from later versions of iostat.  From the man page "The average service time (svctm field) value is meaningless, as I/O statistics are now calculated at block level, and we don't know when the disk driver starts to process a request. For this reason, this field will be removed in a future sysstat version."

Reply:
ack